lib/models/Site.dart

```diff
@@ -11,7 +11,8 @@ import 'StaticHosts.dart';
 var uuid = Uuid();
 
 class Site {
-  static const platform = MethodChannel('net.defined.mobileNebula/NebulaVpnService');
+  static const platform =
```

Comment:
Looks like your IDE may have taken a large hammer and adjusted much whitespace. I don't find a lot of value in it personally. Happy to discuss further if others have strong feelings.

I'll continue after we resolve these first two

Reply:
I've pulled out most of the flutter formatting which should make salient changes easy to identify. 👍